BraintreeCore/src/test/java/com/braintreepayments/api/PaymentMethodNonceUnitTest.java

```diff
@@ -56,6 +64,8 @@ public void constructor_parsesLocalPaymentPlainJSONObject() throws JSONException
         assertFalse(nonce.isDefault());
         assertEquals(PaymentMethodType.PAYPAL, nonce.getType());
         assertEquals("e11c9c39-d6a4-0305-791d-bfe680ef2d5d", nonce.getString());
+        assertEquals("PayPal", nonce.getTypeLabel());
```

Comment:
Do we need both of these (description and typeLabel)? They look to be the same in most cases. Which ones does Drop In use?

Reply:
They are used in Drop-in in different ways currently - the description is used to display the PayPal email address or Venmo account name on vaulted payment methods and the typeLabel is used to get the display icons (and getType alone won't work for that because it needs to be card brand specific or PayPal or Venmo etc). We are still working on the Drop-in refactoring though, so it's possible we will find a better way to consolidate these, but since we are adding them back as package-private we can always remove in the future if no longer needed.

Reply:
There's a lot of coupling between the core sdk and DropIn in Android. It seems like it naturally evolved that way. We could take a final look to make sure these methods actually provide value to merchants. But we know at least if we leave some stuff out of `v4` we can always add it back in a minor release so we have some options.